frontend/src/app/order-details/order-details.component.ts

```diff
@@ -35,5 +35,14 @@ export class OrderDetailsComponent implements OnInit {
 
     this.dialog.open(TrackingNumberComponent, dialogConfig);
   }
+  
+  releaseOrder(order : number): void{ 
+    this.orderControllerService.releaseOrder(order).subscribe();
+    window.location.reload();
+  }
 
+  claimOrder(order: number):void{
+    this.orderControllerService.assignToMe(order).subscribe();
+    window.location.reload();
```

Comment:
Same as the above, but with other status, and update the 'assignedTo' field to the currently signed in user